tests/unit/core/client/test_ConnectionManager.py

```diff
@@ -213,4 +213,18 @@ def test_stop(self):
         self.assertEqual(0, self.connection_manager.num_peer_connections())
         self.assertEqual(None, self.connection_manager._next_manage_call)
 
+    @defer.inlineCallbacks
+    def test_slow_replying_server(self):
```

Comment:
its okay for tests to have a long name - perhaps: `test_when_server_is_slow_then_connection_is_closed(self)`